tests/fixture/e2e/helpers.go

```diff
@@ -183,7 +183,12 @@ func SendEthTransaction(tc tests.TestContext, ethClient ethclient.Client, signed
 		return true
 	}, DefaultTimeout, DefaultPollingInterval, "failed to see transaction acceptance before timeout")
 
-	require.Equal(types.ReceiptStatusSuccessful, receipt.Status)
```

Comment:
This seems to imply that we would only want to issue successful transactions. The gas consumption in this PR actually expects the transaction to fail.